system/jlib/jstats.cpp

```diff
@@ -1005,6 +1005,7 @@ class CStatisticCollection : public CInterfaceOf<IStatisticCollection>
         {
             Statistic * next = Statistic::deserialize(in, version);
             stats.append(*next);
+            next->Release();
```

Comment:
Super picky, but generally prefer making Owned, in case of exception.  But append() unlikely to except and you could argue that this is more self documenting showing the Release() was intended.